### PR TITLE
sf RNs: May 10

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/sf/README.md [hunk 1]
```diff
@@ -21,12 +21,46 @@ Additional documentation:
...
+
+    * AIScoringModelDefinition
+    * AIScoringModelDefVersion
+    * SkillType
+
+* FIX: The JUnit test results after a successful execution of `project deploy start --junit` no longer include an empty failure tag. 
+ 
+     Many thanks to [Robin Windey](https://github.com/R0Wi) who contributed the fix. We love it, and hope to see more from you and the community! (GitHub issue [#2076](https://github.com/forcedotcom/cli/issues/2076), plugin-deploy-retrieve PR [#610](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/610))
+     
+* FIX: If a deploy encounters a `Socket connection timeout` error, the command now keeps retrying the deploy up to the retry limit. (GitHub issue [#2086](https://github.com/forcedotcom/cli/issues/2086), source-deploy-retrieve PR [#957](https://github.com/forcedotcom/source-deploy-retrieve/pull/957))
```
Comment (mshanemc) [inline releasenotes/sf/README.md]:
```suggestion
* FIX: If a deploy or retrieve encounters a `Socket connection timeout` error, the command now keeps retrying the deploy up to the retry limit. (GitHub issue [#2086](https://github.com/forcedotcom/cli/issues/2086), source-deploy-retrieve PR [#957](https://github.com/forcedotcom/source-deploy-retrieve/pull/957))
```

File: releasenotes/sf/README.md [hunk 2]
```diff
@@ -21,12 +21,46 @@ Additional documentation:
...
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Autocomplete now works on Windows [PowerShell](https://learn.microsoft.com/en-us/powershell/)! Partially type a Salesforce CLI command or flag, then press Tab to autocomplete it or press Tab twice to see all the available commands or flags. Install the feature with these steps:
+
+   1. From a PowerShell window, run `sf autocomplete powershell`. 
+   2. Follow the displayed instructions.
+   3. If autocomplete doesn’t work immediately after installation, run `sf autocomplete --refresh-cache`. Then open a new PowerShell window.
+ 
+* NEW: When the `project generate manifest` command runs into an unknown metadata type, such as from a misspelled metadata file, the command now correctly returns an error. The error includes a handy suggestion for a similar-sounding metadata type that the command _does_ know about. (source-deploy-retrieve PR [#948](https://github.com/forcedotcom/source-deploy-retrieve/pull/948)
```
Comment (mshanemc) [inline releasenotes/sf/README.md]:
```suggestion
* NEW: When the `project generate manifest` command runs into an unknown metadata type, such as from a misspelled metadata file, the error now includes handy suggestions for one or more similar metadata types that the command _does_ know about. (source-deploy-retrieve PR [#948](https://github.com/forcedotcom/source-deploy-retrieve/pull/948)
```

Comment (mshanemc) [inline releasenotes/sf/README.md]:
```suggestion
* NEW: When the `project generate manifest` command runs into an unknown metadata type, such as from a misspelled metadata file, the error now includes handy suggestions for one or more similar metadata types that the command _does_ know about. (source-deploy-retrieve PR [#948](https://github.com/forcedotcom/source-deploy-retrieve/pull/948)
```

File: releasenotes/sf/README.md [hunk 3]
```diff
@@ -21,12 +21,46 @@ Additional documentation:
...
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
-## 1.77.0 (May 10, 2023) [stable-rc]
-
-ANNOUNCEMENT: If you install Salesforce CLI using `npm`, and use Node.js 14 or 16, be aware of these [end-of-life dates](https://github.com/forcedotcom/cli/issues/1985).
+## 1.78.0 (May 17, 2023) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Autocomplete now works on Windows [PowerShell](https://learn.microsoft.com/en-us/powershell/)! Partially type a Salesforce CLI command or flag, then press Tab to autocomplete it or press Tab twice to see all the available commands or flags. Install the feature with these steps:
```
Comment (cristiand391) [inline releasenotes/sf/README.md]: No need to press `tab` twice, that's only needed in bash completion.

**Pull request merged**